### PR TITLE
Adds Tutorial for Remote Staking via XCM (via new personal branch)

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds a tutorial for Remote Staking via XCM.
Feedback and discussion located here: https://github.com/PureStake/moonbeam-docs/pull/480 but https://github.com/PureStake/moonbeam-docs/pull/480 will be closed. 

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tutorials/remote-staking-xcm.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Let's review the parameters passed along with this command:
+
+* The `-w` flag corresponds to the endpoint we’re using to fetch this information
+* The `-a` flag corresponds to your Moonbase relay chain address
+* The `-n` flag corresponds to the encoded form of “westend”, the name of the relay chain that Moonbase relay is based on
+
+The script will return 32-byte and 20-byte addresses. We’re interested in the ethereum-style account - the 20-byte one. Feel free to look up your multilocation derivative account on [Moonscan](https://moonbase.moonscan.io/){target=_blank}. You’ll note that this account is empty. You’ll now need to fund this account with at least 1.1 DEV. As this is the amount that the faucet dispenses, you'll need to make a minimum of two faucet requests or you can always reach out to us on [Discord](https://discord.com/invite/amTRXQ9ZpW){target=_blank} for additional DEV tokens.
+
+## Preparing to Stake on Moonbase Alpha {: #preparing-to-stake-on-moonbase-alpha }
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:44]: Some of the headers are "Preparing" and others are just "Get", I would stick to "Prepare" and "Get" or "Preparing" and "Getting"

File: tutorials/remote-staking-xcm.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+You can also programmatically fetch all of the required staking information. Feel free to skip to the [next section](#generating-the-encoded-call-data) if you'd like.
+
+### Retrieve the List of Candidates {: #retrieve-the-list-of-candidates } 
+
+Prior to staking tokens, you’ll need the address of the collator you want to delegate to. To retrieve the list of collator candidates available in the network, head to the **Developer** tab, click on **Chain State**, and take the following steps:
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:56]: Just realized that these sections are reused. I think we should probably put them into snippets so if something changes we only have to make the change in one place

Comment (eshaben) [reply tutorials/remote-staking-xcm.md:56]: - ### Retrieve the List of Candidates {: #retrieve-the-list-of-candidates } 
- ### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations } 
- ### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations }

File: tutorials/remote-staking-xcm.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+ 1. Choose the pallet to interact with. In this case, it is the **parachainStaking** pallet
+ 2. Choose the state to query. In this case, it is the **selectedCandidates** or **candidatePool** state
+ 3. Send the state query by clicking on the **+** button
+
+Each extrinsic provides a different response:
+
+ - **selectedCandidates** — returns the current active set of collators, that is, the top collator candidates by total tokens staked (including delegations). For example, on Moonbase Alpha it is the top {{ networks.moonbase.staking.max_candidates }} candidates
+ - **candidatePool** — returns the current list of all the candidates, including those that are not in the active set
+
+![Staking Account](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:67]: the images aren't in order

File: tutorials/remote-staking-xcm.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+First and foremost, you’ll need the address of the collator you want to delegate to. To locate it, head to the [Moonbase Alpha Staking dApp](https://apps.moonbeam.network/moonbase-alpha/staking){target=_blank} in a second window. Ensure you’re on the correct network, then press **Select a Collator**. Next to your desired collator, press the **Copy** icon. You’ll also need to make a note of the number of delegations your collator has. The [PS-31 collator](https://moonbase.subscan.io/account/0x3A7D3048F3CB0391bb44B518e5729f07bCc7A45D){target=_blank} shown below has `60` delegations at the time of writing. 
+
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+You can also programmatically fetch all of the required staking information. Feel free to skip to the [next section](#generating-the-encoded-call-data) if you'd like.
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:52]: If they skip ahead to the next section, they don't have their existing delegations, so you may just want to add something here about that

File: tutorials/remote-staking-xcm.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+ 1. Copy the code from the previous snippet and paste it inside the code editor box 
+ 2. (Optional) Click the save icon and set a name for the code snippet, for example, **Get delegation count**. This will save the code snippet locally
+ 3. To execute the code, click on the run button
+ 4. Copy the result as you'll need it when initiating a delegation
+
+![Get existing delegation count](/images/tutorials/remote-staking-via-xcm/xcm-stake-7.png)
+
+## Generating the Encoded Call Data {: #generating-the-encoded-call-data }
+
+Then, head to [Moonbase Alpha Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/accounts){target=_blank}. In order to see the **Extrinsics** menu here, you’ll need to have at least one account accessible in Polkadot.js Apps. If you don’t, create one now. Then, head to the **Developer** tab and Press **Extrinsics**.
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:123]:
```suggestion
Then, head to [Moonbase Alpha Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/accounts){target=_blank}. In order to see the **Extrinsics** menu here, you’ll need to have at least one account accessible in Polkadot.js Apps. If you don’t, create one now. Then, head to the **Developer** tab and press **Extrinsics**. 
```

File: tutorials/remote-staking-xcm.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+const collatorAddress = 'COLLATOR-ADDRESS-HERE'; 
+const collatorInfo = await api.query.parachainStaking.candidateInfo(collatorAddress);
+console.log(collatorInfo.toHuman()["delegationCount"]);
+```
+
+![Get candidate delegation count](/images/tutorials/remote-staking-via-xcm/xcm-stake-6.png)
+
+### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations }
+
+If you've never made a delegation from your address you can skip this section. However, if you're unsure how many existing delegations you have, you'll want to run the following JavaScript code snippet to get `delegationCount` from within [Polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/js){target=_blank}:
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:96]: I know I made a comment about reusing this section, but maybe only part of it and explicitly state here that they need to get the delegations for the multilocation derivative account

**Review by eshaben — CHANGES_REQUESTED** [review]

File: tutorials/remote-staking-xcm.md (hunk 4 above)
Comment (eshaben) [reply tutorials/remote-staking-xcm.md:52]: ah I did see there was something about it in the steps where the existing delegations number is used, so I don't think it's necessary to add something here

File: tutorials/remote-staking-xcm.md (hunk 2 above)
Comment (themacexpert) [reply tutorials/remote-staking-xcm.md:59]: Linking to these sections instead of copying them to keep the tutorial as streamlined as possible

**Comment by themacexpert** [comment]
Should be all set now, the main changes I made were streamlining the tutorial by linking to the JS snippets (fetch the number of existing delegations, etc). vs. copying and pasting them in.

File: tutorials/remote-staking-xcm.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+    {
+      "originType": "SovereignAccount",
+      "requiredWeightAtMost": "40000000000",
+      "call": {
+        "encoded": "0x0c113a7d3048f3cb0391bb44b518e5729f07bcc7a45d000064a7b3b6e00d00000000000000002c01000025000000"
+      }
+
+    }
+}
+```
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:170]: I think adding the associated encoded call data here would be nice

**Review by eshaben — COMMENTED** [review]
One last request! But other than that it looks good!! nice job! i like the subject matter 🙂

**Comment by albertov19** [comment]
One thing we have to consider is the sections we would want this to live under the left hand side navigation menu. We could have sections like Ethereum-API, Substarte-API, XCM. Let's think about these today but I feel it is important. We could also do a bit of trickery to prevent redirects in the future if the sections change

File: tutorials/remote-staking-xcm.md (hunk 7 above)
Comment (themacexpert) [reply tutorials/remote-staking-xcm.md:170]: Added!

**Review by eshaben — APPROVED** [review]
LGTM! ✨ 

@albertov19 do you want to review again before merging?

also I figure we can merge this into the `tutorials` branch now and handle sections/format for the tutorials section later

**Pull request merged**